scss/blocks/tabs.scss

```diff
@@ -258,8 +263,92 @@
   margin-bottom: 0px;
 }
 
-.accordion-body > :nth-child(2) {
+.accordion-body > :nth-child(2),
+.tabs-as-columns .col-sm > :nth-child(2) {
   // 2nd child, because the first one is anchor
-  padding-top: 0px;
+  padding-top: 10px;
   margin-top: 0px;
-}
\ No newline at end of file
+}
+
+.dos {
+  padding-right: 50px;
+}
+
+.dos,
+.donts,
+ul li {
+  line-height: 1.5;
+  list-style: none;
+}
+
+.dos ul li::before {
+  content: '';
+  display: block;
+  background-image: $check-mark;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: left top;
+  height: 28px;
+  width: 28px;
+  position: absolute;
+  left: 0;
+}
+.dos h3::before {
+  content: '';
+  display: block;
+  background-image: $check-bordered;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: left top;
+  height: 30px;
+  width: 30px;
+  position: absolute;
+  left: 0;
+}
+.donts h3::before {
+  content: '';
+  display: block;
+  background-image: $cross-bordered;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 30px;
+  width: 30px;
+  position: absolute;
+  left: 0;
+}
+
+.donts h3 {
+  list-style: none;
+  color: #c13329;
+  font-weight: bold;
+  font-size: 18px;
+  padding-left: 29px;
+  font-weight: bold;
+}
+.dos h3 {
+  list-style: none;
+  padding-left: 29px;
+  color: #509f47;
+  font-weight: bold;
+  font-size: 18px;
+}
+
+.donts ul li::before {
+  content: '';
+  display: block;
+  background-image: $cross;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: left top;
+  height: 28px;
+  width: 28px;
+  position: absolute;
+  left: 0;
+}
```

Comment:
you have a similar block starting on line 284, we can optimise here as well similarly to my suggestion above ⬆️